STYLE.md

```diff
@@ -0,0 +1,323 @@
+❌ You can create a new database for each tenant.
+❌ Creating a new bucket lets you set different permissions for images uploaded by users.
+
+The action always comes first and is in the beginning of the sentence, which makes important steps easier to follow.
+
+If a step is more of a suggestion or is optional, you can intentially use another form to make it easier for users
+to skip and scan a document.
+
+### Paragraphs
+
+Like sentences, important information always comes first.
+This makes it easier to scan through the page.
+
+✅ Clear, important information such as actions come first
+> Store secrets as environment variables in vaults by navigating to **settings** > **security** > **vault**. Your secrets should never be shared. You must ensure data privacy, sharing secrets can compromise security during development.
+
+❌ Unclear, important information is in the middle of the paragraph
+> Security is important in development. That's why you should take care to protect secrets. Secrets should be safely stored as a environment variable in a vault. You can find vaults under **settings** > **security** > **vault**. Don't share this with anyone!
+
+If there are multiple important actions or pieces of information, **break up the paragraph**.
+Even if your paragraph is just one or two sentences, shorter paragraphs are easier to scan.
+
+### Diction
+
+If you're unsure about which word to use to describe a concept, you shuold look for precedence in the following order.
+
+1. Appwrite docs
+2. Appwrite Console
+3. Appwrite API specs
+4. Existing blogs
+5. Follow other products when possible.
+6. If all other avenues are explored, propose a new term. Clearly outline in your PR and come to agreement with the Appwrite team.
+
+### Page structure
+
+Quick starts, tutorials, product docs, and other pages should stick very close to existing examples.
+This applies to tone, structure, and content. Unless no exisitng examples of a page type exist,
+or a page needs to be sufficiently different from existing pages, follow exisitng examples.
+
+If you are proposing a new type of page, discuss an outline in your PR and ask for the Appwrite team's review.
+
+## Code snippets
+
+For quick starts and tutorials, a developer must be able to follow code examples from beginning to end
+easily, and the code example is expected to be runnabel and complete.
+
+This means, you need to include imports, dependencies, and all parts needed to arrive at a functional example.
+
+For concept and journey product pages, still try your best to have complete examples, unless:
+
+1. The example will become opinionated. We avoid opinionated implementation and choices in product pages. Keep them in blogs, quick starts, and tutorials.
+2. The example cannot be given in a complete form cleanly, for example many of the Messaging services's examples cannot be given in complete form because the boiler plate and set up is complex and documented in Android/Swift documentation.
```

Comment:
```suggestion
2. The example cannot be given in a complete form cleanly. For example, many of the Messaging services's examples cannot be given in complete form because the boiler plate and set up is complex and documented in Android/Swift documentation.
```